cmd/validator/validator.go

```diff
@@ -79,7 +79,24 @@ func getFlags() (validatorConfig, error) {
 	reportTypePtr := flag.String("reporter", "standard", "Format of the printed report. Options are standard and json")
 	versionPtr := flag.Bool("version", false, "Version prints the release version of validator")
 	groupOutputPtr := flag.String("groupby", "", "Group output by filetype, directory, pass-fail. Supported for Standard and JSON reports")
-	quietPrt := flag.Bool("quiet", false, "If quiet flag is set. It doesn't print any output to stdout.")
+	quietPtr := flag.Bool("quiet", false, "If quiet flag is set. It doesn't print any output to stdout.")
+
+	flagsEnvMap := map[string]string{
+		"depth":              "CSV_DEPTH",
+		"exclude-dirs":       "CSV_EXCLUDE_DIRS",
+		"exclude-file-types": "CSV_EXCLUDE_FILE_TYPES",
+		"output":             "CSV_OUTPUT",
+		"reporter":           "CSV_REPORTER",
+		"groupby":            "CSV_GROUPBY",
+		"quiet":              "CSV_QUIET",
```

Comment:
Why the `CSV` prefix? Did you mean `CFV`?

Reply:
It was the habit to type "CSV". Sorry about that. I fixed it.